conf/core/AttributeConflation.conf

```diff
@@ -19,6 +18,7 @@
   "review.score.criterion.min.threshold": "0.35",
   "tag.key.criterion.keys": "hoot:multilinestring",
   "tag.merger.default": "hoot::OverwriteTag1Merger",
+  "tag.merger.overwrite.exclude": "source;source:imagery;source:imagery:datetime;source:imagery:id;source:imagery:sensor",
   "way.joiner": "hoot::WayJoinerAdvanced",
```

Comment:
You can make the argument that this list is really specific to Tradecraft's version of Attribute Conflation and wouldn't necessary be a good default value for AC as a whole...not sure the best way to handle that yet.

Reply:
I think I'm ok with this list since Tradecraft is driving Attribute Conflation and its use.

Reply:
yeah, its fine for now.